app/src/main/java/com/asfoundation/wallet/verification/ui/credit_card/VerificationCreditCardActivity.kt

```diff
@@ -45,7 +51,7 @@ class VerificationCreditCardActivity : BaseActivity(), VerificationCreditCardAct
     val isWalletVerified = intent.getBooleanExtra(IS_WALLET_VERIFIED, false)
     val title =
         if (isWalletVerified) R.string.verify_card_title else R.string.verification_settings_unverified_title
-    setTitle(title)
+    setTitle("")
     toolbar()
```

Comment:
is necessary keep this code? if not, I think is better removing it

Reply:
removed 👍 they were redundant with the label on the manifest